### ISSUE TITLE
[crash] Something went wrong

### ISSUE THREAD

**Issue by wilj** [kubeshop/monokle#1848]
**Describe the bug**

I opened Monokle for the first time, clicked to add a cluster, and then while I was looking up my cluster information in another window, this error popped up.

**Steps to reproduce**

I'm using k3d locally, and only get the error if I don't have a cluster created before starting Monokle.

**Stacktrace** 

 ```
Cannot set properties of undefined (setting 'cluster')
The error may be correlated with this previous error:
TypeError: Cannot set properties of undefined (setting 'cluster')
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:331671:21
    at Array.forEach (<anonymous>)
    at Function.unflatten (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:331640:23)
    at mergeConfigs (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:59051:237)
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:58666:4928
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644675:25
    at memoized (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644689:40
    at Object.memoized [as current] (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at Object.checkForUpdates [as onStateChange] (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:634742:48)


 TypeError: Cannot set properties of undefined (setting 'cluster')
The error may be correlated with this previous error:
TypeError: Cannot set properties of undefined (setting 'cluster')
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:331671:21
    at Array.forEach (<anonymous>)
    at Function.unflatten (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:331640:23)
    at mergeConfigs (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:59051:237)
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:58666:4928
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644675:25
    at memoized (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644689:40
    at Object.memoized [as current] (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at Object.checkForUpdates [as onStateChange] (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:634742:48)


    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:331671:21
    at Array.forEach (<anonymous>)
    at Function.unflatten (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:331640:23)
    at mergeConfigs (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:59051:237)
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:58666:4928
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644675:25
    at memoized (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644689:40
    at memoized (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:644583:20)
    at useSelectorWithStoreAndSubscription (file:///opt/Monokle/resources/app.asar/build/bundle.main.js:634709:30)
```

**Comment by devcatalin** [comment]
Hi @wilj , thank you for reporting this!
We are aware of the issue and we're trying to come up with a fix as soon as possible.

**Comment by devcatalin** [comment]
Hi @wilj , we released the `1.7.5` version of Monokle which includes a fix for this error.
You can download it here: https://github.com/kubeshop/monokle/releases/tag/v1.7.5
I'm closing this issue but please let us know if the problem persists after 1.7.5.
Thank you!
